Sorting and Searching/Insertion_Sort/Insertion_Sort.js

```diff
@@ -0,0 +1,39 @@
+//Credit:-GeeksForGeeks
```

Comment:
Apart from this, kindly provide the link to this page/code on GFG.

Reply:
I have made the changes now.